src/origin.js

```diff
@@ -37,7 +37,9 @@ export class Origin {
     if (origin === undefined) {
       PLATFORM.eachModule((key, value) => {
         if (typeof value === 'object') {
+          var isBrowserWindow = typeof window !== 'undefined' && value === window;
```

Comment:
why use `var` in this context? `const` is more appropriate.
also: checking if `value === window` should suffice.

Reply:
well agree about `var`. Though it seems in v1, we always use `let`, so maybe change it to that?
About the check, I think we should have `typeof`, as we want to make sure if this module is used in node (jest test for example), it won't break others.

Reply:
@bigopon regarding the typeof. but you have a strict equality between `value` and `window`, and one line before that you already checked that `typeof` value is `object`. so even without knowing anything about platform etc.. this is an obvious redundancy.

Reply:
Right, missed that line. Thanks!

Reply:
Even if we have already checked that `typeof value === 'object'`, in a node environment the condition `value === window` will throw when `window` is not defined. The `typeof window !== 'undefined'` is there to prevent this as pointed out by @bigopon.
I can correct the `var` to `let`, OK?

Reply:
you are right. my bad.
doesn't aurelia have the PLATFORM abstraction exactly for this type of gotchas?

Reply:
@bigopon @avrahamcool I have now corrected the `var` to `let` (not `const` since this was said to be a convention in v1).
Forgot that the I was editing pre-transpilation code for a second :-)

Reply:
Yeah right... oh well at least we botg missed it. Feel less bad 😅 

Thanks @josundt